README.md

```diff
@@ -56,7 +56,7 @@ Input                     | Description
 `build-options`           | <p>(Optional) MATLAB build options, specified as a list of options separated by spaces. The action supports the same [options](https://www.mathworks.com/help/matlab/ref/buildtool.html#mw_50c0f35e-93df-4579-963d-f59f2fba1dba) that you can pass to the `buildtool` command when running a MATLAB build.<p/><p>**Example:** `build-options: -continueOnFailure`<br/>**Example:** `build-options: -continueOnFailure -skip test`</p>
 `startup-options`         | <p>(Optional) MATLAB startup options, specified as a list of options separated by spaces. For more information about startup options, see [Commonly Used Startup Options](https://www.mathworks.com/help/matlab/matlab_env/commonly-used-startup-options.html).<p/><p>Using this input to specify the `-batch` or `-r` option is not supported.<p/><p>**Example:** `startup-options: -nojvm`<br/>**Example:** `startup-options: -nojvm -logfile output.log`</p>
 
-When you use this action, a file named `buildfile.m` must be in the project root directory. 
+When you use this action, a file named `buildfile.m` must be in the MATLAB current folder or its parent folders.
```

Comment:
Ditto

Reply:
Thanks for your comment, @sameagen-MW. Do you feel the new link and note help clarify things?

Reply:
See comment on the base matlab-actions PR.